pkg/service/volume_service_test.go

```diff
@@ -37,23 +37,35 @@ type VolumeServiceTestSuite struct {
 	tenantFolderService             *FolderService
 }
 
+func checkEnvironmentVariableExists() {
+	if os.Getenv("SDK_TARGET_HOST") == "" ||
+		os.Getenv("SDK_TARGET_USER") == "" ||
+		os.Getenv("SDK_TARGET_USER_PASSWORD") == "" ||
+		os.Getenv("SDK_TARGET_TENANT_USER") == "" ||
```

Comment:
Is it obvious what all these env vars mean and how to set them?  If not, the error message could be expanded and/or documentation added somewhere.

Reply:
hmm should the documentation be added in Readme instead? The error message could be
```
ERROR: Missing one of these environment variables: SDK_TARGET_HOST, SDK_TARGET_USER, SDK_TARGET_PASSWORD
See 'Readme' for usage
```

Reply:
The README file is referring to the examples directory.  The test requirements seem to be different.  They are two different audiences.